### PR TITLE
NO-ISSUE: Remove unnecessary GH workflows and fix sec advisories

### PR DESCRIPTION
**Pull request by ricardozanini**
**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
In this PR, we are removing a few workflows that are not being used anymore. 

**Special notes for reviewers**:
There are a few security issues with these based on GH code scan.

**Additional information (if needed):**
If needed we can always bring prow commands back. Labeler wasn't working at all.

### CODE REVIEW

File: .github/workflows/prow_commands.yml [hunk 1]
```diff
@@ -1,43 +0,0 @@
-# Copyright 2022 The Serverless Workflow Specification Authors
```
Comment (spolti) [inline .github/workflows/prow_commands.yml:1]: why removing these?

Comment (ricardozanini) [reply .github/workflows/prow_commands.yml:1]: not being used, the action is outdated, and GH keeps complaining about permissions. If you know which permissions to give to this pipelines, I can keep them.

**Review by spolti — APPROVED** [review]

File: .github/workflows/prow_commands.yml (hunk 1 above)
Comment (spolti) [reply .github/workflows/prow_commands.yml:1]: I didn't notice any issue related to permissions, I've compared it with another repository that uses this very same prow and the token permissions are the same.

But I have no strong opinion about removing and keeping it :)

/approve

Comment (ricardozanini) [reply .github/workflows/prow_commands.yml:1]: These permissions here:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token

Do you know which?

**Pull request merged**